import-export-cli/integration/apim/apim.go

```diff
@@ -478,8 +478,10 @@ func (instance *Client) AddAPI(t *testing.T, api *API, username string, password
 	json.NewDecoder(response.Body).Decode(&apiResponse)
 
 	t.Cleanup(func() {
-		instance.Login(username, password)
-		instance.DeleteAPI(apiResponse.ID)
+		if doClean {
```

Comment:
Can we move this if block to before the `t.Cleanup()` is defined? That way we can avoid registering the Cleanup function altogether if doClean == true. Currently doClean == true results in an empty Cleanup() being registered which does nothing.

Reply:
Sure will do the changes where applicable including for API Products as well